### PR TITLE
Bugfix/#1878

### PR DESCRIPTION
**Pull request by romanovskyroma**

### CODE REVIEW

File: src/app/component/eco-news/components/news-list/news-list-gallery-view/news-list-gallery-view.component.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 $bp-smallest: 575px;
 $bp-phone: 767px;
-$bp-tablet: 991px;
+$bp-tablet: 1024px;
```
Comment (luchward) [inline src/app/component/eco-news/components/news-list/news-list-gallery-view/news-list-gallery-view.component.scss:3]: I suppose it should be 1023px for the media query

File: src/app/component/eco-news/components/news-list/news-list-gallery-view/news-list-gallery-view.component.scss [hunk 2]
```diff
@@ -1,6 +1,6 @@
 $bp-smallest: 575px;
```
Comment (luchward) [inline src/app/component/eco-news/components/news-list/news-list-gallery-view/news-list-gallery-view.component.scss:1]: I told you dont't touch these components until my pull request will be merged. You waste your time.

Comment (romanovskyroma) [reply src/app/component/eco-news/components/news-list/news-list-gallery-view/news-list-gallery-view.component.scss:1]: I saw your pull request and econews gallery display correct. So I will delete this pull request.

**Comment by romanovskyroma** [comment]
this bug was fixed in another pull request.  [#664](https://github.com/ita-social-projects/GreenCityClient/pull/664)

**Pull request closed without merging**